### ISSUE TITLE
Quickstart code missing a step?

### ISSUE THREAD

**Issue by frackham** [KevReed/PlantUml.Net#1]
**TLDR:** Looks like I'm missing either a reference or a path, but can't see what needs setting?

Context:
- Plantuml works locally, version test works.
- dot test works, Graphiviz version 2.38.0
- Plantuml version 1.2018.05
- Java version 1.8.0_144-b01
- OS: Win 10.
- Can generate a diagrams using local jar file.
- Visual Studio Community 2017 15.2,  C# console app.
- References added to project (nuget). Builds.

Using the example quickstart code... 
```
var factory = new RendererFactory();
var renderer = factory.CreateRenderer(new PlantUmlSettings());

var bytes = renderer.Render("Bob -> Alice : Hello", OutputFormat.Png);
File.WriteAllBytes("out.png", bytes);

```
in the console Main method, I got:
![capture](https://user-images.githubusercontent.com/2105204/39970408-4adc4d34-56da-11e8-82fb-bf669557213c.PNG)

Have tried with breaking down the code and setting the plantumlsettings directly, but still hit an issue:
![capture](https://user-images.githubusercontent.com/2105204/39970425-b43cc74a-56da-11e8-9102-3a612cb52d41.PNG)

**Comment by frackham** [comment]
Can't edit again for some issue, but apologies as I think I'm missing something obvious!

**Comment by KevReed** [comment]
Hi,

It looks like I was not respecting the JavaPath setting correctly, this has been updated in 1.0.22
let me know if this fixes the issue.

**Comment by frackham** [comment]
That's got me there, thanks. 

Just FYI, I had a further issue that I could resolve myself from there - I don't think that LocalPlantUmlPath supports paths with spaces in the name? 

Got a fail on the first version here:
plantumlSettings.LocalPlantUmlPath = "C:\\Alt VS\\PlantUml\\plantuml.jar"
and not on...
plantumlSettings.LocalPlantUmlPath =  Path.Combine(@"C:\", "PlantUml", "plantuml.jar");

(Tried with Path.Combine on first version too).

Thanks for the speedy response! :)

**Comment by KevReed** [comment]
Thanks for reporting.
